### PR TITLE
Slack Integration Improvements

### PR DESCRIPTION
**Pull request by benzuzu**
### The following options have been added to the Slack Integration dropdown menu

- Send Academic For Profit Email
- Send Academic Domain Clarification Email
- Send Use Case Clarification Email
- Send Duplicate User Email
- Send Rejection Email
- Send Alumni Rejection Email

Clicking on these options prompts an input modal to appear where the email can be modified and sent directly from Slack.

This fixes https://github.com/oncokb/oncokb/issues/2654

### CODE REVIEW

File: src/main/resources/templates/mail/alumniEmailAddress.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <title th:text="#{email.reject.title(${license})}"></title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <link rel="shortcut icon" th:href="@{|${baseUrl}/favicon.ico|}" />
+</head>
+<body>
+<p th:text="'Dear ' + ${user.firstName} + ' ' + ${user.lastName} + ','"></p>
+
+<p th:text="'Thank you for your interest in the ' + ${license} + ' license for OncoKB.'"></p>
+
+<p>We noticed that you registered your account under an alumni email address. Unfortunately, we are unable to approve accounts registered under alumni addresses for any OncoKB license.</p>
```
Comment (MoriahNissan) [inline src/main/resources/templates/mail/alumniEmailAddress.html:13]: We noticed that you registered your account using an alumni email address. Unfortunately, we are unable to approve accounts registered with alumni addresses for any OncoKB license.

File: src/main/resources/templates/mail/alumniEmailAddress.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<p>Please contact us if this is a mistake.</p>
```
Comment (MoriahNissan) [inline src/main/resources/templates/mail/alumniEmailAddress.html:15]: Please re-register using an email address from your current institution. If you think you got this message in error, please contact us.

File: src/main/resources/templates/mail/clarifyDuplicateUser.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <link rel="shortcut icon" th:href="@{|${baseUrl}/favicon.ico|}" />
+</head>
+<body>
+<p th:text="'Dear ' + ${user.firstName} + ' ' + ${user.lastName} + ','"></p>
+
+<p th:text="'Thank you for your interest in the ' + ${license} + ' license for OncoKB.'"></p>
+
+<p>We noticed that you already have a registered OncoKB account. Users are generally not allowed to have more than one OncoKB account.</p>
+
+<p>Therefore, could you please provide your reasons as to why you require multiple accounts? Thank you.</p>
```
Comment (MoriahNissan) [inline src/main/resources/templates/mail/clarifyDuplicateUser.html:15]: If you have forgotten your login credentials or if you believe you may have received this message in error, please contact us. Thank you.

File: src/main/resources/templates/mail/clarifyUseCase.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+<body>
+<p th:text="'Dear ' + ${user.firstName} + ' ' + ${user.lastName} + ','"></p>
+
+<p th:text="'Thank you for your interest in the ' + ${license} + ' license for OncoKB.'"></p>
+
+<p>
+  Upon reviewing your license request, we were not able to fully understand and approve of the use case you provided.
+</p>
+
+<p>Therefore, could you please further explain your use case of OncoKB? Thank you.</p>
```
Comment (MoriahNissan) [inline src/main/resources/templates/mail/clarifyUseCase.html:17]: Therefore, could you please further clarify your use case of OncoKB? Thank you.

File: src/main/resources/templates/mail/rejectionEmail.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  <title th:text="#{email.reject.title(${license})}"></title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <link rel="shortcut icon" th:href="@{|${baseUrl}/favicon.ico|}" />
+</head>
+<body>
+<p th:text="'Dear ' + ${user.firstName} + ' ' + ${user.lastName} + ','"></p>
+
+<p th:text="'Thank you for your interest in the ' + ${license} + ' license for OncoKB.'"></p>
+
+<p>Unfortunately, we are currently unable to approve your account with the use case provided.</p>
```
Comment (MoriahNissan) [inline src/main/resources/templates/mail/rejectionEmail.html:13]: Unfortunately, we are currently unable to approve your account with the use case you provided.

File: src/main/resources/templates/mail/rejectionEmail.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+<p>Please feel free to contact us to contest this decision.</p>
```
Comment (MoriahNissan) [inline src/main/resources/templates/mail/rejectionEmail.html:15]: If you feel you have received this message in error, please contact us.

**Review by zhx828 — APPROVED** [review]

**Pull request merged**